liveview-android/src/main/java/org/phoenixframework/liveview/ui/phx_components/LiveView.kt

```diff
@@ -1,30 +1,99 @@
 package org.phoenixframework.liveview.ui.phx_components
 
+import android.net.Uri
+import android.util.Log
 import androidx.compose.foundation.layout.fillMaxSize
 import androidx.compose.material.MaterialTheme
 import androidx.compose.material.Surface
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.DisposableEffect
 import androidx.compose.runtime.collectAsState
 import androidx.compose.runtime.getValue
+import androidx.compose.runtime.remember
 import androidx.compose.ui.Modifier
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.rememberNavController
+import androidx.navigation.navArgument
 import org.phoenixframework.liveview.domain.LiveViewCoordinator
 import org.phoenixframework.liveview.ui.theme.LiveViewTestTheme
 
+private const val TAG = "LiveView"
+private const val PHX_LIVE_VIEW_ROUTE = "phxLiveView"
+private const val ARG_ROUTE = "route"
+
 @Composable
-fun LiveView(liveViewCoordinator: LiveViewCoordinator) {
-    val state by liveViewCoordinator.backStack.collectAsState()
+fun LiveView(
+    url: String,
+) {
+    // The WebSocket URL is the same of the HTTP URL, so we just copy the HTTP URL changing the schema (protocol)
+    val webSocketBaseUrl = remember(url) {
+        Uri.parse(url).buildUpon().scheme("ws").build().toString()
```

Comment:
This should switch between `ws` and `wss` based on `http`/`https` of the LiveView URL.